qiita_db/metadata_template/util.py

```diff
@@ -251,13 +251,13 @@ def looks_like_qiime_mapping_file(fp):
     some other different column.
     """
     first_line = None
-    with qdb.util.open_file(fp, newline=None) as f:
+    with qdb.util.open_file(fp, newline=None, errors='replace') as f:
         first_line = f.readline()
     if not first_line:
         return False
 
     first_col = first_line.split()[0]
-    return first_col == '#SampleID'
+    return first_col in ('#SampleID', b'#SampleID')
```

Comment:
why not either always open in raw, or cast the value on line 255 to bytes? Technically, I think this is a qiime1 mapping file too, right?

Reply:
Good point, will try to address this. Yup, qiime1 mapping file but the issue comes cause the users are in Windows with some weird Excel configuration that added some unicode that wasn't parsable.

Reply:
My issue was on creating and then reading the file. Decided to simplify things by just having the file in the repo and reading it as it's done in other tests ...